### DISCUSSION TITLE
Validation as

### DISCUSSION THREAD

**Discussion by azamspeare** [livewire/livewire#7871] [1. Help] [answered]
How can validate name as according to localization?
My current approach doesn't work?

```
#[Validate('required|string', as: trans('fields.permissions.description'))]
public ?string $description = null;
```

What should I do to get translation for property

**Comment by joshhanley** [comment, marked as answer]
@Abduazam you can't call functions in PHP attributes (see [PHP docs](https://www.php.net/manual/en/language.attributes.syntax.php)), only literal values or constant expressions.

But Livewire has the translation helper built in for you, so it will auto translate the string, so just call `as 'fields.permissions.description'` or you can opt-out of translation if you don't want it, see https://livewire.laravel.com/docs/validation#opting-out-of-localization
